src/plugin/analytics/Controller/Administration/SecurityLogController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Claroline\AnalyticsBundle\Controller\Administration;
```

Comment:
Should be in `Controller\ApiNew\Log` IMO.

Reply:
Done, thx :)